benchmark/cluster/vgg16/fluid_pserver.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           value: "/workspace"
         - name: PADDLE_INIT_PORT
           value: "30236"
+        - name: PORT
```

Comment:
Should change `vgg16_fluid.py` to use `PADDLE_INIT_PORT` rather than add a new env.

Reply:
Done.